src/index.html

```diff
@@ -4,9 +4,444 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald:wght@700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
+
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <header class="header">
+    <a
+      href="#Home"
+      class="header__home"
+    >
+      <img
+        src="images/logo.png"
+        alt="Home"
+      >
+    </a>
+    <nav class="header__nav">
+      <ul class="header__list">
+        <li class="header__item">
+          <a
+            href="#apple"
+            class="header__link header__link--active is-active"
+          >
+            Apple
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+            href="#samsung" class="header__link"
+          >
+            Samsung
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+             href="#smartphones" class="header__link"
+          >
+             Smartphones
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+            href="#laptops&computers"
+            class="header__link"
+            data-qa="nav-hover"
+          >
+            Laptops & Computers
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+            href="#gadgets" class="header__link"
+          >
```

Comment:
```suggestion
          <a href="#gadgets" class="header__link">
```

In general, if you place two attributes in one line you can place the whole tag in one line. It looks better.